### PR TITLE
DOCSP-39174 Add Restart Task Substep to Windows Instructions

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Adding a restart command to the windows install instructions.
- Added a note to the all install instructions that if you started the application before moving the odbc drivers into the driver folder you must restart.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-39174/installation/install-on-a-local-machine/install-windows/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-39174/installation/install-on-a-local-machine/install-ubuntu/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-39174/installation/install-on-a-local-machine/install-rhel/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-39174/installation/install-on-a-local-machine/install-mac/

## JIRA

https://jira.mongodb.org/browse/DOCSP-39174

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=664cbcd3cf5bec1c29671fac

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/fact-restart-migrator-oracle-mysql-drivers.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.. note::
+
+   If you started Relational Migrator before copying the driver files
```
Comment (ltran-mdb2) [inline source/includes/fact-restart-migrator-oracle-mysql-drivers.rst:3]: [suggestion] we could probably make this present tense

```suggestion
   If you start Relational Migrator before you copy the driver files 
```

**Review by ltran-mdb2 — APPROVED** [review]
LGTM mod a small suggestion! Thanks @ianf-mongodb !

**Comment by ianf-mongodb** [comment]
Hi @tomhollander this came in as a support case this week so I wanted to get you to take a look in case you had anything to add.

**Review by tomhollander — CHANGES_REQUESTED** [review]
Thanks for proactively improving the docs here. However I think we can do a bit better here.

1. There's no reason why the format of the instructions should vary from one OS to the other. You've added a note box to 3/4 of the install instructions, there's no reason to treat Windows differently.
2. The note is a bit misleading as the need to restart isn't dependent on which database is being used. It would be more accurate to say something like "Relational Migrator must be restarted in order to load any drivers added to your Drivers folder."
3. These topics are related to installation, and if you actually follow them as written then the app won't be running at the time the drivers are copied over. While we should also accommodate users who copy drivers after starting the app, the steps should still be consistent.

So overall, across all 4 operating systems, I think the final step should be to _Start_ the application, but then you can add a note saying to _Restart_ if it was already running when you added the drivers.

**Review by ianf-mongodb — COMMENTED** [review]
Hi @tomhollander after reviewing I see your point and aligning the slight variations on each instruction set makes sense. To that end I added a final step to windows for double clicking the exe to start the add with the additional context note for the non-default drivers.

**Review by tomhollander — APPROVED** [review]

**Pull request merged**